### PR TITLE
Bugfix/18626/owner transfer multiselect

### LINKED ISSUE #18626: [Transfer ownership] bad click behavior on selecting from suggestions
Clicking on one of the suggested transfer receivers seems to accept the entry, but there's no feedback on the input field about it:

size and confirmation

## Steps to reproduce

1. select a file to transfer
2. type in the user to transfer to
3. see autocomplete showing some suggestions
4. click on a suggestion
5. **see no feedback from the input field**
6. see the final button accepting the target user
7. click somewhere into blank
8. see the input field accepting the target user

### PR DESCRIPTION
**Pull request by juliusknorr**
Fixes https://github.com/nextcloud/server/issues/18626

Zhe Multiselect component was placed inside of a label element, so the click event to select the entry was bubbling to the parent label and therefore triggering the open again. Moving the multiselect outside of the label makes sure that it will be closed on select.

Apart from that I also added a commit to respect the minSearchStringLength configuration option and fetch some users as soon as the page is loaded, so that the user input is not empty if it is opened for the first time.

### CODE REVIEW

File: apps/files/src/components/TransferOwnershipDialogue.vue [hunk 1]
```diff
@@ -95,6 +96,9 @@ export default {
 			loadingUsers: false,
 			selectedUser: null,
 			userSuggestions: {},
+			config: {
+				minSearchStringLength: parseInt(OC.config['sharing.minSearchStringLength'], 10) || 0,
```
Comment (gary-kim) [inline apps/files/src/components/TransferOwnershipDialogue.vue]: :+1:

**Review by gary-kim — APPROVED** [review]

**Review by rullzer — APPROVED** [review]
awesome thanks!

**Comment by juliushaertl** [comment]
stable18 https://github.com/nextcloud/server/pull/19291

**Pull request merged**